cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java

```diff
@@ -234,27 +241,15 @@ public void drop(String name) throws Exception {
     deleteConsumerConfigurations(queueName);
   }
 
-  @Override
-  public void upgrade(String tableName, Properties properties) throws Exception {
-    AbstractHBaseDataSetAdmin dsAdmin = new DatasetAdmin(tableName, hConf, tableUtil, properties);
-    try {
-      dsAdmin.upgrade();
-    } finally {
-      dsAdmin.close();
-    }
-  }
-
-  private void drop(byte[] tableName) throws IOException {
-    HBaseAdmin admin = getHBaseAdmin();
-    if (admin.tableExists(tableName)) {
-      admin.disableTable(tableName);
-      admin.deleteTable(tableName);
+  private void drop(HBaseAdmin admin, TableId tableId) throws IOException {
```

Comment:
this method shouldn't need admin, no? ``getHBaseAdmin()`` should be available here?